bitcoind-tests/tests/test_csfs.rs

```diff
@@ -178,16 +179,48 @@ pub fn test_desc_satisfy(cl: &ElementsD, testdata: &TestData, desc: &str) -> Vec
             let sig = secp.sign_schnorr_with_aux_rand(&msg, keypair, &aux_rand);
             Some(sig)
         }
+
+        fn lookup_price_oracle_sig(
+                &self,
+                pk: &bitcoin::XOnlyPublicKey,
+                time: u64,
+            ) -> Option<(secp256k1::schnorr::Signature, i64, u64)> {
```

Comment:
In 5e533b71c8904f8ad2e0f6ec7ec50f33e3ecd13a:

nit: formatting is weird. (In general, we can format this crate in CI, I believe, since we are doing so for rust-miniscript already.)

Reply:
Yep, will do in followup